core/services/relay/evm/mercury/transmitter.go

```diff
@@ -141,7 +145,16 @@ func (mt *mercuryTransmitter) Start(ctx context.Context) (err error) {
 			return err
 		}
 		mt.wg.Add(1)
-		go mt.runloop()
+
+		transmissions, err := mt.orm.GetTransmitRequests()
+		if err != nil {
+			return err
+		}
+		for _, transmission := range transmissions {
+			mt.queue.Push(transmission.Req, transmission.ReportCtx)
+		}
```

Comment:
There is a faster way to do this. How about reading in with `created_at DESC` order then initializing the queue with this to start with. You can call `heap.Init` should be very efficient.